modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-major.adoc

```diff
@@ -0,0 +1,359 @@
+= Anypoint MQ Connector 3.x Migration Guide
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint MQ Connector version 3.0 includes new features, such as simplified configuration, standardized endpoints, and non-blocking functionality.
+
+=== Global Configuration
+
+In the previous version of the Anypoint MQ Connector, the `config` element included parameters, as shown in this example:
+
+[source,xml,linenums]
+----
+<anypoint-mq:default-subscriber-config
+    name="Anypoint_MQ_Default_subscriber"
+    acknowledgementMode="IMMEDIATE"
+    acknowledgementTimeout="10000" <!-- (1) -->
+    maxRedelivery="2"              <!-- (2) -->
+    pollingTime="2000"
+    fetchSize="5"                  <!-- (3) -->
+    fetchTimeout="5000"            <!-- (3) -->
+    frequency="2000">              <!-- (3) -->
+  <anypoint-mq:connection          <!-- (4) -->
+    clientId="${client.id}"
+    clientSecret="${client.secret}"/>
+  <anypoint-mq:circuit-breaker     <!-- (5) -->
+    circuitName="InvoiceSettings"
+    onErrorTypes="JMS:TIMEOUT"
+    errorsThreshold="10" tripTimeout="1"
+    tripTimeoutUnit="MINUTES" />
+</anypoint-mq:default-subscriber-config>
+----
+
+In this version, the `config` element for the MQ Connector now contains only the connection settings. For example:
+
+[source,xml,linenums]
+----
+<anypoint-mq:config name="Anypoint_MQ_Config">
+    <anypoint-mq:connection
+        clientId="${client.id}"
+        clientSecret="${client.secret}" />
+</anypoint-mq:config>
+----
+
+The following table describes the `config` parameter differences.
+
+[%header,cols="20,40,40"]
```

Comment:
Should be `[%header,cols="20a,40a,40a"]` as you're using asciidoc conventions in your table